README.md

```diff
@@ -30,7 +30,7 @@ Here are some examples of how you could leverage `digitalocean-cloud-controller-
 
 ### do not modify DO load-balancers manually
 
-When creating load-balancers through CCM (via `LoadBalancer`-typed Services), it is important that you **must not change the DO load-balancer configuration manually.** Such changes will eventually be reverted by the reconciliation loop built into CCM. One exception are load-balancer names which can be changed (see also [the documentation on load-balancer ID annotations](/docs/getting-started.md#load-balancer-id-annotations)).
+When you are creating load-balancers through CCM (via `LoadBalancer`-typed Services), it is very much important that you **must not change the DO load-balancer configuration manually.** Such changes will eventually be reverted by the reconciliation loop built into CCM. There will be one exception which is load-balancer names. It can be changed (see also [the documentation on load-balancer ID annotations](/docs/getting-started.md#load-balancer-id-annotations)).
```

Comment:
I think "it is very much important" isn't grammatically correct -- we should omit "much" here.

Reply:
please Merge my PR